src/components/FilterBar/FilterBar.tsx

```diff
@@ -33,18 +34,30 @@ export let FilterBar = withStore('distribution', 'measure', 'standard')(({ store
       </DropDownMenu>
     </div>
 
-    <div className='Filter'>
+    <div className='Filter -FixedWidthBig'>
       <span>Measure</span>
       <DropDownMenu
         className='DropDownMenu -Compact'
         onChange={(e, i, value) => store.set('measure')(value)}
         value={store.get('measure')}
       >
         {Array.from(TIME_DISTANCES).map(([miles, mins]) =>
-          <MenuItem key={miles} value={miles} primaryText={`${miles} miles / ${mins} min`} />
+          <MenuItem
+            key={miles}
+            value={miles}
+            primaryText={getMeasureText(miles, mins, store.get('standard'))}
+          />
         )}
       </DropDownMenu>
     </div>
 
   </Paper>
 )
+
+function getMeasureText(miles: Measure, mins: number, standard: Standard) {
```

Comment:
Can you briefly explain what this function is doing in a comment?

Reply:
Done.